### PR TITLE
Ui fixed

### PR DESCRIPTION
**Pull request by dokun1**
You now have a UI that pops up every time you open the app to enter a 4 digit GUID. The UI looks like so:

img_0421

I've also enhanced the settings screen to allow for a default location spoof, as well as the host URL and the openWhisk token. Tap the API Connect logo on the main screen 4 times to get to this UI:

img_0422 png

Only downside is that, when default location isn't turned on, the location isn't reflected in the map view on the main controller - let me know if you think that's necessary. I didn't add it because I figured we are circumventing natural behavior - best to stick with what's expected visually.

### CODE REVIEW

**Comment by dokun1** [comment]
worth mentioning that the second it takes to "validate" the guid is a fake second for UX purposes only.

File: LogisticsWizardMobile/LogisticsWizardMobile.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -17,6 +17,8 @@
 		535A6CD61E786B92008472D3 /* Trip.swift in Sources */ = {isa = PBXBuildFile; fileRef = 535A6CD51E786B92008472D3 /* Trip.swift */; };
 		535A6CD81E788F5A008472D3 /* SettingsController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 535A6CD71E788F5A008472D3 /* SettingsController.swift */; };
 		535C15B11E785CFF00171705 /* LocationManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 535C15B01E785CFF00171705 /* LocationManager.swift */; };
+		53F135271E940A1F0014DFF5 /* PinInput.swift in Sources */ = {isa = PBXBuildFile; fileRef = 53F135261E940A1F0014DFF5 /* PinInput.swift */; };
+		53F135291E940A810014DFF5 /* GUIDInputViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 53F135281E940A810014DFF5 /* GUIDInputViewController.swift */; };
```
Comment (joesepi) [inline LogisticsWizardMobile/LogisticsWizardMobile.xcodeproj/project.pbxproj:21]: Is this your xcode project file? If so, I would assume you would want that in your `.gitignore` and not in the repo. Same goes for `.DS_Store` files too.

**Review by joesepi — COMMENTED** [review]
Besides the little nit picks, LGTM

File: LogisticsWizardMobile/LogisticsWizardMobile.xcodeproj/project.pbxproj (hunk 1 above)
Comment (dokun1) [reply LogisticsWizardMobile/LogisticsWizardMobile.xcodeproj/project.pbxproj:21]: So `.DS_Store` files should be in gitignore already.

As for `.xcodeproj` files, it's a fun school of thought to get into. For this, I'm not using any crazy package management systems like Cocoapods that corrupt it and make the file unrecognizable, and you can't really run a mobile app without that file. 

Server side swift, it's generally a good idea to not have it committed to the repo. 

They can definitely be a pain when dealing with merge conflicts, but shouldn't be much of an issue here.

**Comment by dokun1** [comment]
Also FWIW I only tagged everyone to review so you can see Swift code :-D

File: LogisticsWizardMobile/LogisticsWizardMobile.xcodeproj/project.pbxproj (hunk 1 above)
Comment (joesepi) [reply LogisticsWizardMobile/LogisticsWizardMobile.xcodeproj/project.pbxproj:21]: If `.DS_Store` was in your `.gitignore`, there would not be any in this PR. :)

And I defer to your expertise on the project file. Whatever makes sense in Swift land.

**Comment by svennam92** [comment]
Should the PR be against this repo instead? :) https://github.com/strongloop/acme-freight-mobile

Also in reference to your comment:
> Only downside is that, when default location isn't turned on, the location isn't reflected in the map view on the main controller - let me know if you think that's necessary.

Note sure if the above comment makes sense to me - when the default location **isn't** turned on, the location isn't reflected? Did you mean when the default location **is** turned on, the location doesn't show up in the map view?

I would think that the map view should correspond with the current longitude/latitude when default location is turned off.

**Comment by dokun1** [comment]
Drats! Let's merge this in first, then I'll swap the remote after we get it in.

I'll clarify what I meant with the location bit: in the UI of the map view, your current location is always reflected, regardless of the settings. The only difference is when you actually register a trip. If the default location is off, it will process your actual coordinates for the registration. If it is on, it will use the provided latitude and longitude in the settings for registering the trip.

**Pull request merged**